warehouse/migrations/versions/81f9f9a60270_add_gitlab_oidc_models.py

```diff
@@ -0,0 +1,85 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Add GitLab OIDC models
+
+Revision ID: 81f9f9a60270
+Revises: be62a4cd76e3
+Create Date: 2024-01-16 17:46:16.443395
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+revision = "81f9f9a60270"
+down_revision = "be62a4cd76e3"
+
+
+def upgrade():
+    op.execute(
+        """
+        INSERT INTO admin_flags(id, description, enabled, notify)
+        VALUES (
+            'disallow-gitlab-oidc',
+            'Disallow the GitLab OIDC provider',
+            FALSE,
```

Comment:
```suggestion
            TRUE,
```

I think we want this to be disabled by default, similar to:

https://github.com/pypi/warehouse/blob/09ea793f35a14bc29e5f05fe93486a8ce2cd615c/warehouse/migrations/versions/9a0ed2044b53_add_activestate_oidc_publisher.py#L69-L80

Reply:
fixed